sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/api/GeoVelocityDataResolver.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.extension.siddhi.execution.geovelocity.api;
+
+import org.wso2.extension.siddhi.execution.geovelocity.internal.exception.GeoVelocityException;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+
+/**
+ * Interface for the Geo-velocity based data on username and city.
+ */
+public interface GeoVelocityDataResolver {
```

Comment:
Format all the comments

Reply:
Fixed in https://github.com/wso2/samples-is/pull/265